pkgs/by-name/gi/git-blame-someone-else/package.nix

```
@@ -0,0 +1,44 @@
+{ callPackage
+, lib
+, stdenv
+, fetchurl
+, nixos
+, testers
+, versionCheckHook
+, git-blame-someone-else
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "git-blame-someone-else";
+  version = "0";
+
+  src = fetchurl {
+    url = "https://github.com/jayphelps/git-blame-someone-else/archive/refs/heads/master.zip";
+    hash = "sha256-jZkUKv2SV28wsM18tCqNxoCZmLxdYH2Idh9RLibH2yA=";
+  };
+
+  doCheck = true;
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+
+  # Give git-blame-someone-else some install checks for testing purpose.
+  postInstallCheck = ''
+    stat "''${!outputBin}/bin/${finalAttrs.meta.mainProgram}"
+  '';
+
+
+  meta = with lib; {
+    description = "Blame someone else for your bad code.";
+    longDescription = ''
+      A simple script to spoof a git commit as someone else. Created as a joke, not recommended for production use.
+    '';
+    homepage = "https://github.com/jayphelps/git-blame-someone-else";
+    license = licenses.mit;
+    maintainers = ["https://github.com/WGrav01"];
+    mainProgram = "git-blame-someone-else";
+    platforms = platforms.all;
+  };
```

```suggestion
  meta = {
    description = "Blame someone else for your bad code.";
    longDescription = ''
      A simple script to spoof a git commit as someone else.
    '';
    homepage = "https://github.com/jayphelps/git-blame-someone-else";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [wgrav01];
    mainProgram = "git-blame-someone-else";
    platforms = lib.platforms.all;
  };
```
Same as in the old `package.nix`

---

I was sloppy and lost that commit somehow, sorry

---

Also for context the original creator of the script put that in their disclaimer so I figured it would be best I put it in the long description, should it stay then?

---

> Also for context the original creator of the script put that in their disclaimer so I figured it would be best I put it in the long description, should it stay then?

oh i didn't see that change, seems good to have there.

> I was sloppy and lost that commit somehow, sorry

git is not known to be user friendly, so shit happens.

---

Pulling over ssh was hanging so i ran `git push -f` out of impatience. I'm going to bed but the nixfmt test is failing despite my running nixfmt on it and finding nothing in the git diff, any ideas on what's happening there?